dockerhub-readme/templates/base.md.j2

```diff
@@ -195,6 +195,10 @@ Notebooks can be found in the following directories within the {{ repo_rapids_ve
 
 For a full description of each notebook, see the [README](https://github.com/rapidsai/notebooks/blob/branch-{{ repo_rapids_version }}/README.md) in the notebooks repository.
 
+### Building on top of RAPIDS images
+
+All RAPIDS images use `conda` as their package manager, and all RAPIDS packages (including source-built) are installed under the `rapids` environment name. If you want to build on top of the RAPIDS images (such as using `FROM`), then it is important to include `source activate rapids` at the start of all `RUN` commands in your `Dockerfile`. Without this, the docker build context will not have access to the RAPIDS libraries, as it uses the `base` environment by default. Examples of this can be found in our own Dockerfiles, which can be found in the [RAPIDS Docker Repository](https://github.com/rapidsai/docker) on GitHub.
```

Comment:
```suggestion
All RAPIDS images use `conda` as their package manager, and all RAPIDS packages (including source-built) are available in the `rapids` conda environment. If you want to extend RAPIDS images (such as using `FROM`), then it is important to include `source activate rapids` at the start of all `RUN` commands in your `Dockerfile`. Without this, the docker build context will not have access to the RAPIDS libraries, as it uses the `base` environment by default. Examples of this can be found in our own Dockerfiles, which can be found in the [RAPIDS Docker Repository](https://github.com/rapidsai/docker) on GitHub.
```

Reply:
couple of semantic suggestions.